examples/v1alpha3/hyperband-example.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       - name: "eta"
         value: "3"
       - name: "r_l"
-        value: "9"
+        value: "2"
```

Comment:
Is this needed?

Reply:
Yeah, or the validation will fail because the hyperband algorithm will validate parallelTrialCount according to the r_l

Reply:
How was it working in v1alpha2?

Reply:
In e2e test, we will set the parallel trial count to 2. Thus we will get the error `parallel trial count is less than 9` bacause of the r_l. In v1alpha2 we do not have e2e test for hyperband. Thus we do not have the problem.

Reply:
Got it. Thanks for explanation.